### PR TITLE
Endpoints

### PR DESCRIPTION
**Pull request by jamesquillen**

### CODE REVIEW

File: app/controllers/apartments_controller.rb [hunk 1]
```diff
@@ -3,4 +3,15 @@ def index
         apartments = Apartment.all
         render json: apartments
     end
+
+    def create
+        apartment = Apartment.create
+        render json: apartment
+    end
```
Comment (elysemontano) [inline app/controllers/apartments_controller.rb:10]: I don't think this method should be on this PR as this is not part of the Trello card.

File: spec/requests/apartments_spec.rb [hunk 2]
```diff
@@ -1,4 +1,57 @@
...
+                    user_id: user.id
+        )
+        
+
+        post '/apartments'
+        apartment = Apartment.first
+
+        delete "/apartments/#{apartment.id}"
+
+        expect(response).to have_http_status(200)
```
Comment (elysemontano) [inline spec/requests/apartments_spec.rb:54]: Is there any additional test we can add in here that will verify the apartment got deleted?

**Review by elysemontano — CHANGES_REQUESTED** [review]
Please see comments in apartments controller and consider expanding the test for your destroy method.

File: app/controllers/apartments_controller.rb (hunk 1 above)
Comment (jamesquillen) [reply app/controllers/apartments_controller.rb:10]: When we ran the test for the destroy method, it failed with a message saying there was no create method. When we added the create method the test passed.

File: spec/requests/apartments_spec.rb (hunk 2 above)
Comment (jamesquillen) [reply spec/requests/apartments_spec.rb:55]: We added an additional expect section to expect that after deletion the list of apartments should be zero objects long.

**Review by elysemontano — APPROVED** [review]

**Pull request merged**